cms/templates/home_page.html

```diff
@@ -130,13 +130,13 @@ <h2 class="course-card-title">
   </div>
 {% endif %}
 {% if show_home_page_video_component and page.video_component_title %}
-<div class="home-page-video-row d-flex justify-content-center row">
-  <div class="home-page-video-container">
-    <div class="home-page-video-description">
+<div class="home-page-video-row d-flex justify-content-center container">
+  <div class="home-page-video-container row">
+    <div class="home-page-video-description d-flex flex-column col-12 col-xl-5">
       <h3>{{ page.video_component_title }}</h3>
-      <p>{{ page.video_component_description|richtext }}</p>
+      {{ page.video_component_description|richtext }}
     </div>
-      <div class="home-page-video-wrapper">
+      <div class="home-page-video-wrapper d-flex justify-content-center col-12 col-xl-7">
```

Comment:
Will `d-flex` and `justify-content-center` be inherited from  `<div class="home-page-video-row d-flex justify-content-center container">`

Reply:
I think this comment still persists.